src/explore-education-statistics-admin/src/pages/admin-dashboard/components/__tests__/DraftReleasesTable.test.tsx

```diff
@@ -0,0 +1,521 @@
+    expect(within(row3cells[1]).getByText('Draft')).toBeInTheDocument();
+    expect(
+      within(row3cells[2]).getByRole('link', { name: 'Edit Release 1' }),
+    ).toHaveAttribute(
+      'href',
+      '/publication/publication-1/release/release-1/summary',
+    );
+    expect(
+      within(row3cells[2]).queryByRole('button', { name: /View issues/ }),
+    ).not.toBeInTheDocument();
+    expect(
+      within(row3cells[2]).queryByRole('button', {
+        name: 'Cancel amendment for Release 1',
+      }),
+    ).not.toBeInTheDocument();
+    expect(
+      within(row3cells[2]).queryByRole('link', {
+        name: 'View original for Release 1',
+      }),
+    ).not.toBeInTheDocument();
+
+    // In review
+    const row4cells = within(rows[3]).getAllByRole('cell');
+    expect(within(row4cells[0]).getByText('Release 3')).toBeInTheDocument();
+    expect(within(row4cells[1]).getByText('In Review')).toBeInTheDocument();
+    expect(
+      within(row4cells[2]).getByRole('link', { name: 'Edit Release 3' }),
+    ).toHaveAttribute(
+      'href',
+      '/publication/publication-1/release/release-3/summary',
+    );
+    expect(
+      within(row4cells[2]).queryByRole('button', { name: /View issues/ }),
+    ).not.toBeInTheDocument();
+    expect(
+      within(row4cells[2]).queryByRole('button', {
+        name: 'Cancel amendment for Release 3',
+      }),
+    ).not.toBeInTheDocument();
+    expect(
+      within(row4cells[2]).queryByRole('link', {
+        name: 'View original for Release 3',
+      }),
+    ).not.toBeInTheDocument();
+
+    expect(within(rows[4]).getByRole('columnheader')).toHaveTextContent(
+      'Publication 2',
+    );
+
+    // Amendment
+    const row6cells = within(rows[5]).getAllByRole('cell');
+    expect(row6cells[0]).toHaveTextContent('Release 2');
+    expect(within(row6cells[0]).getByText('Release 2')).toBeInTheDocument();
+    expect(
+      within(row6cells[1]).getByText('Draft Amendment'),
+    ).toBeInTheDocument();
+    expect(
+      within(row6cells[2]).getByRole('link', { name: 'Edit Release 2' }),
+    ).toHaveAttribute(
+      'href',
+      '/publication/publication-2/release/release-2/summary',
+    );
+    expect(
+      within(row6cells[2]).getByRole('button', {
+        name: 'Cancel amendment for Release 2',
+      }),
+    ).toBeInTheDocument();
+    expect(
+      within(row6cells[2]).getByRole('link', {
+        name: 'View original for Release 2',
+      }),
+    ).toBeInTheDocument();
+    expect(
+      within(row6cells[2]).queryByRole('button', { name: /View issues/ }),
+    ).not.toBeInTheDocument();
+
+    expect(within(rows[6]).getByRole('columnheader')).toHaveTextContent(
+      'Publication 3',
+    );
+
+    // Draft
+    const row8cells = within(rows[7]).getAllByRole('cell');
+    expect(within(row8cells[0]).getByText('Release 4')).toBeInTheDocument();
+    expect(within(row8cells[1]).getByText('Draft')).toBeInTheDocument();
```

Comment:
No... Release 4 isn't an amendment, only Release 2 is.

Reply:
Ah yeh sorry, should have looked a little closer! Why do we have this additional draft release then? Seems like it would be more worthwhile to check against an in review amendment instead?

Reply:
yeah good point, I've changed it to be an in review amendment as suggested.